pkg/controller/core.oam.dev/v1alpha2/application/application_controller_test.go

```diff
@@ -848,7 +839,7 @@ var _ = Describe("Test Application Controller", func() {
 		Expect(k8sClient.Delete(ctx, app)).Should(BeNil())
 	})
 
-	It("Test rollout trait all related definition features", func() {
+	PIt("Test rollout trait all related definition features", func() {
```

Comment:
Do we need those tests in the future?

Reply:
Let's keep it for now. If the legacy rollout API is removed in code completely, we can remove this test.